apps/zui/src/js/initializers/init-monaco.ts

```diff
@@ -0,0 +1,192 @@
+import {loader} from "@monaco-editor/react"
+
+const primitiveTypes = [
+  "uint8",
+  "uint16",
+  "uint32",
+  "uint64",
+  "int8",
+  "int16",
+  "int32",
+  "int64",
+  "float16",
+  "float32",
+  "float64",
+  "bool",
+  "string",
+  "duration",
+  "time",
+  "bytes",
+  "ip",
+  "net",
+  "type",
+  "null",
```

Comment:
The Zed types in zed-js (e.g, TypeOfUint8) have a name property. Would it make sense to use those name properties here?

Reply:
Yes, that could work.